pkg/app/api/grpcapi/web_api.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/pipe-cd/pipe/pkg/crypto"
 	"github.com/pipe-cd/pipe/pkg/datastore"
 	"github.com/pipe-cd/pipe/pkg/git"
-	"github.com/pipe-cd/pipe/pkg/insight"
+	insightfilestore "github.com/pipe-cd/pipe/pkg/insight/insightstore"
```

Comment:
nit: Remove "insightfilestore".
I think "insightstore" is good enough and as I see we are using that name at other places too.